base_layer/core/src/validation/helpers.rs

```diff
@@ -127,3 +126,20 @@ pub fn check_achieved_and_target_difficulty<B: BlockchainBackend>(
     }
     Ok(())
 }
+
+pub fn is_stxo<T: BlockchainBackend>(db: &T, hash: HashOutput) -> Result<bool, ValidationError> {
```

Comment:
Should we not put this in the chain_storage rather?

Reply:
As in add it to BlockchainDatabase? I do agree that validators should not really care about how is_stxo is found

Reply:
yes

Reply:
We'd need to add to the `BlockchainBackend` trait (validators only know about the trait (which is probably not correct). So I vote for not doing that in this PR and making that part of a refactor for validators. At least this "helper" is confined to validators - so you know that it only belongs there